ionoscloud/resource_dbaas_redisdb_replicaset_test.go

```diff
@@ -33,6 +35,47 @@ func TestAccDBaaSRedisDBReplicaSetBasic(t *testing.T) {
 					resource.TestCheckResourceAttr(constant.DBaaSRedisDBReplicaSetResource+"."+constant.DBaaSReplicaSetTestResource, replicaSetVersionAttribute, replicaSetVersionValue),
 				),
 			},
+			{
+				Config: redisDBReplicaSetDataSourceMatchID,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDBaaSRedisDBReplicaSetExists(constant.DBaaSRedisDBReplicaSetResource+"."+constant.DBaaSReplicaSetTestDataSourceByID, &replicaSet),
```

Comment:
@avirtopeanu-ionos you added the check for the `location` only for `MatchByName` configuration, please also add it here.